### PR TITLE
Update ErrorMessage component UI

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update ErrorMessage UI

Screenshot 2024-03-21 at 15 05 23

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/app/debugger/car-trial/CarTrialDebuggerForm.tsx [hunk 1]
```diff
@@ -70,22 +70,7 @@ export const CarTrialDebuggerForm = () => {
           </Space>
         </Space>
       </form>
-      {state?.errors?.generic && (
-        <ErrorCard errorCode={'VALIDATION_FAILED'} errorMessage={state.errors.generic[0]} />
-      )}
+      <ErrorMessages errors={state?.errors?.generic} />
```
Comment (Youakeem) [inline apps/store/src/app/debugger/car-trial/CarTrialDebuggerForm.tsx:73]: Maybe try with more than one error to see how it looks like?

Comment (gustaveen) [reply apps/store/src/app/debugger/car-trial/CarTrialDebuggerForm.tsx:73]: You mean like this? @Youakeem 
Screenshot 2024-03-21 at 15 35 25

Comment (Youakeem) [reply apps/store/src/app/debugger/car-trial/CarTrialDebuggerForm.tsx:73]: Yeah, sure! We might wanna get some design feedback on it later but I think it looks good!

**Review by Youakeem — APPROVED** [review]
Great job, man!

**Comment by gustaveen** [comment]
### Merge activity

* **Mar 21, 10:48 AM EDT**: @gustaveen started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4086).

**Pull request merged**